src/decorators/is-date.spec.ts

```diff
@@ -29,12 +34,39 @@ describe('IsDate', () => {
         Result.err('date is not formatted as `yyyy-mm-dd`')
       );
     });
+
+    it('transforms to date formatted string', async () => {
+      const dateToUse = new Date();
+      const obj = new TestSerialize();
+      obj.date = dateToUse;
+
+      expect(await output(obj)).toStrictEqual({ date: dateToUse.toISOString().slice(0, 10) });
+    });
+
+    it('pass through when field is undefined', async () => {
+      const obj = new TestSerialize();
+
+      expect(await output(obj)).toStrictEqual({});
+    });
+
+    it('pass through value if no serialize', async () => {
+      const dateToUse = new Date();
+      const obj = new Test();
+      obj.date = dateToUse;
+
+      expect(await output(obj)).toStrictEqual({ date: dateToUse });
+    });
   });
 
   describe('single date-time', () => {
     class Test {
       @IsDate({ format: 'date-time' })
-      date!: Date;
+      date?: Date;
+    }
+
+    class TestSerialize {
+      @IsDate({ format: 'date-time', serialize: true })
```

Comment:
pls add a test case without `serialize` prop, not sure it's clear how it works from test cases

Reply:
Sure, I'll add some additional test cases. I also don't think `serialize` is the best name for a property, so I'll look into changing it